module-cheatsheets/scRNA-seq-cheatsheet.md

```diff
@@ -10,50 +10,49 @@
 
 ### Base `R`
 
-Read the Base `R` documentation [**here**](https://www.rdocumentation.org/packages/base/versions/3.5.1).
+Read the Base `R` documentation [**here**](https://rdrr.io/r/).
 
 |Library/Package|Piece of Code|What it's called| What it does|
 |---------------|-------------|----------------|-------------|
-| Base `R`| [`rowSums()`](https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/rowSums)| Row sums|Calculates sums for each row|
-| Base `R`| [`colSums()`](https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/colSums)| Column sums| Calculates sums for each column|
-| Base `R`| [`t()`](https://www.rdocumentation.org/packages/base/versions/3.5.1/topics/t)  |Transpose| Returns the transpose of a matrix or data frame|
-| Base `R`| [`prcomp()`](https://www.rdocumentation.org/packages/stats/versions/3.5.1/topics/prcomp)| Principal Components Analysis|Executes a principal components analysis on specified matrix or data frame|
-| Base `R`| [`<-function(x) { <code> }`](http://adv-r.had.co.nz/Functions.html) | Function | Creates a function that would take the defined parameters as input and execute the commands within the curly braces  |
+| Base `R`| [`rowSums()`](https://rdrr.io/r/base/colSums.html)| Row sums|Calculates sums for each row|
+| Base `R`| [`colSums()`](https://rdrr.io/r/base/colSums.html)| Column sums| Calculates sums for each column|
+| Base `R`| [`t()`](https://rdrr.io/r/base/t.html)  |Transpose| Returns the transpose of a matrix or data frame|
+| Base `R`| [`prcomp()`](https://rdrr.io/r/stats/prcomp.html)| Principal Components Analysis|Executes a principal components analysis on specified matrix or data frame|
+| Base `R`| [`<-function(x) { <code> }`](https://adv-r.hadley.nz/functions.html) | Function | Creates a function that would take the defined parameters as input and execute the commands within the curly braces  |
 
 
 ### `ggplot2`
 
-Read the `ggplot2` documentation [**here**](https://ggplot2.tidyverse.org/reference/).
+Read the `ggplot2` documentation [**here**](https://ggplot2.tidyverse.org/).
```

Comment:
First, this is a general thing to come back to for consistency.  

I much prefer presenting the overall landing page for the documentation, sans `reference`, because it allows them to hop anywhere in the ggplot2 docs without being strictly presented with an _overwhelming_ list of functions, most of which will be unfamiliar to them. Maybe we need a third opinion on this one..

Reply:
Middle ground:  we can have it all!

> Read the `ggplot2` package documentation [**here**](https://ggplot2.tidyverse.org/), as well as an overall reference for `ggplot2` functions [**here**](https://ggplot2.tidyverse.org/reference/index.html).

Reply:
> Just noting here that you didn't implement the same changes as in the intro here, but I think that is fine.

Can't hurt, will do. 

> Maybe just add geom_vline to the geom_hline section from the intro and remove it from the single cell doc?

Makes sense to me!

Reply:
I've decided to also delete the final description bit that suggests `geom_vline()` and friends can _only_ be used on top of another geom.